### PR TITLE
Potentially better landing page

### PR DESCRIPTION
**Pull request by MaddyUnderStars**
Replaces the old sections with a features list, with ( imo ) clearer wording and markings.
Some of the wording may not be the best, feel free to correct

#54

### CODE REVIEW

**Comment by MaddyUnderStars** [comment]
localhost_3000_
screenshot for the lazy

**Comment by MaddyUnderStars** [comment]
localhost_3000_
moved links

**Comment by MaddyUnderStars** [comment]
oh the screenshot broke. you get the point, though

**Review by TheArcaneBrony — APPROVED** [review]
looks fine to me

**Comment by MaddyUnderStars** [comment]
@erkinalp thoughts? I don't have rights to merge on this repo.

File: components/features.vue [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+
+				<div class="feature incomplete">
+					<h3>Voice and Video</h3>
+					<p>Voice and Video support on Fosscord is in active development, but may take a while longer.</p>
+					<a class="moreInfo" href="https://github.com/fosscord/fosscord-server/pull/472">Tracker</a>
+				</div>
+
+				<div class="feature">
+					<h3>Much more!</h3>
+					<p>This is by <i>no</i> means a complete list of features, planned or completed, that the team
```
Comment (erkinalp) [inline components/features.vue:81]:
```suggestion
					<p>This is <i>by no means</i> a complete list of features, planned or completed, that the team
```

File: components/features.vue [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+					<a class="moreInfo" href="https://docs.fosscord.com/server/database/">Database documentation</a>
+				</div>
+
+				<div class="feature done">
+					<h3>Separable backend components</h3>
+					<p>
+						Fosscord-server is made up of a few different components: api, gateway, cdn, webrtc. Each can be
+						run as separate services, giving instance owners more control over how their system operates. If
+						you wish, you can run each on separate machines, behind load balances with many instances of
+						each component.
```
Comment (erkinalp) [inline components/features.vue:41]: users might ask why it is a monorepo when components may be run separately, needs explanation

File: components/features.vue (hunk 1 above)
Comment (TheArcaneBrony) [reply components/features.vue:81]: I agree with this change, while a small change, I feel like it would improve UX a little

**Review by TheArcaneBrony — APPROVED** [review]

**Review by FSG-Cat — APPROVED** [review]

**Review by erkinalp — COMMENTED** [review]
reasoning for monorepo structure not explained clearly

**Review by erkinalp — APPROVED** [review]

**Comment by MaddyUnderStars** [comment]
@CanadaHonk Hey, sorry for the ping, but would you mind checking out what you think of the above as well? I would appreciate any feedback you have

**Review by erkinalp — CHANGES_REQUESTED** [review]
Hey, looks good to me, except that reasoning for the monorepo is still missing

**Comment by MaddyUnderStars** [comment]
> Hey, looks good to me, except that reasoning for the monorepo is still missing

Yeah I removed that section because its not really a feature to be advertised. Do you think that reasoning should still be there? Would you fit it in as another feature ( which I'd rather not do as it'd break the 3x3 grid we have going on ) or replace an existing one? Personally I think an explanation for why its a mono repo would be better fit for the documentation

**Comment by erkinalp** [comment]
@MaddyUnderStars OK, in docs FAQ then.

**Review by erkinalp — APPROVED** [review]

**Comment by MaddyUnderStars** [comment]
@erkinalp @TheArcaneBrony If we're happen with this, can someone merge for me? I don't have write perms here

**Comment by erkinalp** [comment]
@MaddyUnderStars neither do i

**Pull request merged**